example/android/link-assets-manifest.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Have your tried using expo's google fonts? https://docs.expo.dev/guides/using-custom-fonts/#using-a-google-font
Inter font is even shown in the example.
That way there is no need for including ttf in native side, it should be easier.

Reply:
I thought about it but that requires me to load the font using useFonts() in any component, and I wasn't sure where to load it. I tried doing that in the main App.tsx component but that resulted with a bunch of errors since the font wasn't loaded in time to be used for style declarations in the fonts/fonts.ts, which is independed from the App.tsx.

However if you have any tips on how to load a font when it is being used outside of the react components, I'd love to try it out.

Reply:
Sure! In another project we use a splash screen (`expo-splash-screen` and `preventAutoHideAsync` function). So while we load the font and other assets we just show the splashscreen and after loading we show the app with `expo-splash-screen`'s `hideAsync`. We have the splashscreen anyway in the designs.

Reply:
You have this even described here: https://docs.expo.dev/guides/using-custom-fonts/#waiting-for-fonts-to-load